spec/requests/api/people_controller_index_v2_spec.rb

```diff
@@ -47,6 +47,14 @@
       let(:query) { '?filter[prison_number]=G3239GV,GV345VG' }
       let(:import_from_nomis) { instance_double('People::ImportFromNomis', call: nil) }
 
+      let!(:people) do
+        [
+          create(:person, prison_number: 'G3239GV'),
```

Comment:
Would it be worth creating some test cases for other identifiers too?

Reply:
Done.